src/rotation.zig

```diff
@@ -507,13 +548,40 @@ pub const RotationModes = struct {
 		pub const id: []const u8 = "stairs";
 		var modelIndex: u16 = 0;
 
-		fn subBlockMask(x: u1, y: u1, z: u1) u8 {
+		inline fn subBlockMask(x: u1, y: u1, z: u1) u8 {
 			return @as(u8, 1) << ((@as(u3, x)*2 + @as(u3, y))*2 + z);
 		}
-		fn hasSubBlock(stairData: u8, x: u1, y: u1, z: u1) bool {
+		inline fn hasSubBlock(stairData: u8, x: u1, y: u1, z: u1) bool {
```

Comment:
These aren't used at runtime in the rotateZ function anymore, so I don't see a reason anymore to mark them inline.